languages/swift/exercises/concept/structs-and-classes/.docs/after.md

```diff
@@ -0,0 +1,298 @@
+[Structs and classes][structs-and-classes] are two of the primary building blocks of Swift programming. They are both means of grouping together related data and functions into self-contained units of functionality. All of the basic types that you have been working with up to this point, such as `Int`, `Double`, and `Character` as well as collection types like `Array`, `String`, and `Dictionary` are actually structs. And when you define a struct or class, you are defining a new type to be used within Swift, just as you used those types.
```

Comment:
When seeing, that big parts are exact copies of the introduction, I am tempted to skip that part completely and might oversee the additional info below.

What about putting the text into 2 parts: 
## Facts from this exercise (or sth. similar)

## Additional information
... followed by Value types v. reference types and further

Reply:
> When seeing, that big parts are exact copies of the introduction, I am tempted to skip that part completely and might oversee the additional info below.

I think you're looking at this from the standpoint of the student who just finished the exercise. The `after.md` document though should be seen as a reference document that students can also return to later. In fact, the concepts will likely have their own pages and thus the contents should be independent from the exercise or the introduction document.